### ISSUE TITLE
InsecureRequestWarning flooding logs

### ISSUE THREAD

**Issue by GrigoriOH** [trunneml/homeassistant-vorwerk#9]
First off, many thanks for your work! Got my VR200 implemented into HA with ease thanks to it 👍 

But after setting everything up, the _core_ logs are being flooded with:

> `  warnings.warn(
> /usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'nucleo.ksecosys.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings`

Short lookup on this url shows vorwerk-cloud@neatorobotics.com as owner of the self signed certificate in use.

From all I understand, an unverified HTTPS request is sent towards ksecosys.com as the verification of the selfsigned certificate is not possible. Am I on track?

**Comment by trunneml** [comment]
Could you attach the full stack trace or at least a bigger part of the log?

**Comment by GrigoriOH** [comment]
After disabling homeassistant-vorwerk and restarting, this is how the Supervisor->Core log looks and stays like:

> [s6-finish] sending all processes the TERM signal. 
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2021-10-01 16:16:21 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration vorwerk which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-10-01 16:16:21 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant

Just after enabling homeassistant-vorwerk, this appears right below these entries:

> /usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'nucleo.ksecosys.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'nucleo.ksecosys.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'nucleo.ksecosys.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
[... and so on]

Could not find a stack trace, though. Sorry for not being able to deliver more.

EDIT:
I tried gathering more info with debug logs, but interestingly the warning just shows in the "Supervisor"->"System"->"Core" **_log_** and not in the actual home-assistant.log **_file_**
